prime/config/config.yaml

```diff
@@ -196,5 +199,24 @@ logging:
     - type: console
       layout:
         type: json
+        customFieldNames:
+          # GCP Stack-driver logging needs "level" to be named as "severity"
+          level: severity
+      filterFactories:
+        # Hiding duplicate WARN and ERROR logs since they are logged again with separate console appender
+        - type: level-filter-factory
+          level: WARN
+          onMatch: DENY
+        - type: level-filter-factory
+          level: ERROR
+          onMatch: DENY
+    # This is to send logs with WARN and above level to GCP Error reporting.
+    - type: console
+      # If this threshold is changed, also change the levels in filterFactories above.
+      threshold: WARN
+      layout:
+        type: json
+        additionalFields:
```

Comment:
This is sweet but obscure.  It would be nice if this was documented e.g. by pointing with an url into the drop wizard documentation